benchmarks/src/main/kotlin/com/google/samples/apps/nowinandroid/interests/InterestsActions.kt

```diff
@@ -35,7 +38,12 @@ fun MacrobenchmarkScope.goToInterestsScreen() {
 fun MacrobenchmarkScope.interestsScrollTopicsDownUp() {
     device.wait(Until.hasObject(By.res("interests:topics")), 5_000)
     val topicsList = device.findObject(By.res("interests:topics"))
-    device.flingElementDownUp(topicsList)
+    if (topicsList != null) {
+        // TODO: Ensure topics are availble.
+        device.flingElementDownUp(topicsList)
+    } else {
+        Log.w(TAG, "No topics found, can't scroll during baseline profile generation.")
```

Comment:
I would suggest failing the test as well. Given benchmark is a test, it would be hard to figure out if the test is doing the right thing. WDYT?

Reply:
Failing now.